code/datums/status_effects/debuffs/debuffs.dm

```diff
@@ -1099,6 +1099,18 @@
 	UnregisterSignal(owner, COMSIG_ATOM_UPDATE_OVERLAYS)
 	owner.update_icon()
 
+/datum/status_effect/ineffective_thermoregulation
```

Comment:
This status effect should give the owner a trait when applied and remove said trait when also removed.